### PR TITLE
Add download button

### PR DESCRIPTION
**Pull request by Mridul97**
Made changes to add a 'download' button next to the 'Add Step' button.

### CODE REVIEW

File: examples/demo.js [hunk 1]
```diff
@@ -26,7 +26,12 @@ window.onload = function() {
   sequencer.loadImage("images/tulips.png", ui.onLoad);
 
   $("#addStep select").on("change", ui.selectNewStepUi);
-  $("#addStep button").on("click", ui.addStepUi);
+  $("#addStep #add-step-btn").on("click", ui.addStepUi);
+  $('#addStep #download-btn').click(function() {
+    $('img:last()').trigger( "click" );
```
Comment (jywarren) [inline examples/demo.js]: Cool, does this work for you?

Comment (Mridul97) [reply examples/demo.js]: Yes, this works for me!

**Comment by jywarren** [comment]
Ok, awesome. Merging!

**Comment by jywarren** [comment]
Oh wait I think it needs a rebase, can you try?

**Comment by Mridul97** [comment]
I have rebased the branch. Please take a look. Thanks!

**Comment by Mridul97** [comment]
I have rebased the branch. Please take a look.

**Comment by jywarren** [comment]
Awesome!!! Great work 👍

**Pull request merged**